lib/Bio/EnsEMBL/DataCheck/Checks/ControlledMetaKeys.pm

```diff
@@ -36,22 +36,27 @@ use constant {
 
 sub tests {
   my ($self) = @_;
-
   my $species_id = $self->dba->species_id;
   my $group = $self->dba->group;
-
   my $sql = qq/
     SELECT meta_key, COUNT(*) FROM meta
     WHERE species_id = $species_id OR species_id IS NULL
     GROUP BY meta_key
   /;
+
   my $helper = $self->dba->dbc->sql_helper;
   my %meta_keys = %{ $helper->execute_into_hash(-SQL => $sql) };
 
+  #check target site is main / new and select mandatory metakeys 
+  my $filter_metakeys = '';
+  if (defined $self->target_site){
+    $filter_metakeys = " AND target_site like '\%".$self->target_site."\%' ";
```

Comment:
Another question, sorry I missed it last time, why escaping the `%` in the string?

Reply:
I don't think they will hurt, but I don't see them doing anything in this context.

Reply:
with in double quote it interpolates as hash, escaped this character.